src/main/java/techreborn/blockentity/storage/item/StorageUnitBaseBlockEntity.java

```diff
@@ -222,6 +265,11 @@ public ItemStack processInput(ItemStack inputStack) {
 	}
 
 	public boolean isSameType(ItemStack inputStack) {
+		if (getLocked()) {
+			System.out.printf("+++ SUB#isSameType: locked: [%s]\n", lockedItem);
```

Comment:
Just log with debug log level

Reply:
Sorry, I left all this stuff in because it's not meant to go into a formal PR.

Reply:
It is absolutely fine to have extended logging. Might help with debugging in future. Just log with debug level